WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -2133,3 +2133,37 @@ extension WooAnalyticsEvent {
         }
     }
 }
+
+// MARK: - Free Trial
+//
+extension WooAnalyticsEvent {
+    enum FreeTrial {
+        enum Keys: String {
+            case source
+            case currentPlan = "current_plan"
+        }
+
+        enum Source: String {
+            case banner
+            case upgradesScreen = "upgrades_screen"
+        }
+
+        enum Plan: String {
+            case freeTrial = "ecommerce-trial-bundle-monthly"
+        }
+
+        static func freeTrialUpgradeNowTapped(source: Source) -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .freeTrialUpgradeNowTapped, properties: [Keys.source.rawValue: source.rawValue])
+        }
+
+        static func planUpgradeSuccess(source: Source, plan: Plan) -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .planUpgradeSuccess, properties: [Keys.source.rawValue: source.rawValue,
+                                                                          Keys.currentPlan.rawValue: Plan.freeTrial.rawValue])
```

Comment:
I see you're going to update this tracking in #9064, so ignore this if it's part of what you'll be changing there:

* Do you mean to use `Plan.freeTrial.rawValue` (instead of `plan.rawValue`) for this property's value?
* Do we mean to track the free trial as the current plan after a successful plan upgrade? I expected that this would track the plan I upgraded to (the Commerce plan).

Reply:
Oops, what an oversight! Good catch!